control/rlocus.py

```diff
@@ -80,6 +88,7 @@ def root_locus(sys, kvect=None, xlim=None, ylim=None, plotstr='-', Plot=True,
     PrintGain: boolean (default = True)
         If True, report mouse clicks when close to the root-locus branches,
         calculate gain, damping and print
+    plotstr: string that declare of the rlocus (see matplotlib)
```

Comment:
Can you add details to this description? E.g.,

    format string for rendering the root loci. Interpretation of the string is defined by matplotlib.pyplot.plot.